dev/RadioButtons/RadioButtons.idl

```diff
@@ -14,8 +14,8 @@ unsealed runtimeclass RadioButtons : Windows.UI.Xaml.Controls.Control
     Windows.Foundation.Collections.IVector<Object> Items{ get; };
     Windows.UI.Xaml.DataTemplate ItemTemplate;
 
-    Windows.UI.Xaml.DependencyObject ContainerFromItem(Object item);
-    Windows.UI.Xaml.DependencyObject ContainerFromIndex(Int32 index);
+    Windows.UI.Xaml.UIElement ContainerFromItem(Object item);
```

Comment:
Consider not having ContainerFromItem to avoid comparing objects. If you dont have it, then the app dev can do that lookup and pass in the index to ContainerFromIndex. Also, it is a bit confusing because there is no container :) so perhaps the naming needs to change.

Reply:
There sort of is a container, if the items source contains anything but a radio button the factory makes a radio button with that object as its content.  I'd love to remove the API though, especially cause right now it doesn't actually work :)

---
In reply to: [347398385](https://github.com/microsoft/microsoft-ui-xaml/pull/1623#discussion_r347398385) [](ancestors = 347398385)

Reply:
The API does work after responding to PR feedback. However I think you're right we should consider if we want this.

---
In reply to: [347514532](https://github.com/microsoft/microsoft-ui-xaml/pull/1623#discussion_r347514532) [](ancestors = 347514532,347398385)